org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/JavaSearchTests.java

```diff
@@ -4503,5 +4507,62 @@ public void testBug383908() throws CoreException {
 		deleteProject("P");
 	}
 }
+/*
+ * Test that we can find methods called in an anonymous type from a jar,
+ * if the primary type contains multiple anonymous types.
+ * https://github.com/eclipse-jdt/eclipse.jdt.core/issues/375
+ */
+public void testAnonymousTypeMethodReferenceJarSearchGh375() throws Exception {
```

Comment:
Looking at this, https://git.eclipse.org/c/jdt/eclipse.jdt.core.git/diff/org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/JavaSearchBugs8Tests.java?id=f06de4b999ef266d99070d6e6ee7f897f16a1d0e, we could create the necessary test files and archive them on-the-fly. We don't have to commit a jar and a project. WDYT @iloveeclipse ?